docs/source/configuration/header-propagation.mdx

```diff
@@ -81,37 +81,38 @@ Enables you to add custom headers to requests going to a specific subgraph. Thes
 Here's a complete example showing all the possible configuration options in use:
 
 ```yaml title="router.yaml"
-headers:
-  # Header rules for all subgraphs
-  all:
-  # Propagate matching headers
-  - propagate:
-      matching: ^upstream-header-.*
-  # Propagate matching headers
-  - propagate:
-      named: "some-header"
-      default: "default-value"
-      rename: "destination-header"
-  # Remove the "x-legacy-account-id" header
-  - remove:
-      named: "x-legacy-account-id"
-  # Remove matching headers
-  - remove:
-      matching: ^x-deprecated-.*
-  # Insert the 'my-company' header
-  - insert:
-      name: "my-company"
-      value: "acme"
-  # Subgraph-specific header rules
-  subgraphs:
-    products:
-      # Calls to the products subgraph have the "router-subgraph-name" header set to `products`.
-      - insert:
-          name: "router-subgraph-name"
-          value: "products"
-    accounts:
-      # Calls to the accounts subgraph have the "router-subgraph-name" header set to `accounts`.
-      - insert:
-          name: "router-subgraph-name"
-          value: "accounts"
+plugins:
```

Comment:
ditto

Reply:
it looks like the rest of the team didn't notice so maybe i m just confused ^^

Reply:
No it's a new change I just committed. It's just, AFAIK the plugins made by apollo are all prefixed by `apollo.` isn't it ? Maybe I missed a discussion.

Reply:
Ok reverted. I didn't know about the feature with apollo plugins

Reply:
All our plugins are either "apollo.<blah>" or "experimental.<blah>". We've changed config processing so that if a plugin name begins with "apollo.", then you don't put it in the "plugins" section you put it at config root and miss out the "apollo." part of the name. (Somewhat confusingly, or maybe not, it still works if you put it in the plugins section as "apollo.<blah>".